.travis.yml

```diff
@@ -10,17 +10,15 @@ before_script:
 jobs:
   include:
 - stage: "API smoke test"
+      name: "DEV1"
+      if: type != api OR env(TEST_JOB) == "dev1"
+    - script: node_modules/.bin/newman run tests/API_contract_tests.postman_collection.json -e tests/DEV1.postman_environment.json --env-var "loginEmail=$DEV1_USER_EMAIL" --env-var "loginPw=$DEV1_USER_PASSWORD" --env-var "marketoClientId=$MARKETO_CLIENT_ID" --env-var "marketoClientSecret=$MARKETO_CLIENT_SECRET" --env-var "marketoApi=$MARKETO_MUNCHKIN_API" --env-var "clinicEmail=$DEV1_CLINIC_EMAIL" --env-var "clinicPw=$DEV1_CLINIC_PASSWORD"
```

Comment:
Yaml formatting is broken here - I suggest you remove all hyphens from all attributes nested under `stage` and  then add one before each `name` attribute. This way it will be more easy to see spot items in the "stage" array. For example:
```
- stage: "API smoke test"
    - name: "dev1"
       if: type != api OR env(TEST_JOB) == "dev1"
       script: "..."
    - name: "qa1"
       if: type != api OR env(TEST_JOB) == "qa1"
       script: "..."
```